CHANGES.md

```diff
@@ -65,6 +57,9 @@
 - Fix crash when decoding dune-package for libraries with `(include_subdirs
   qualified)` (#10269, fixes #10264, @emillon)
 
+- Backport #10327: fix a regression in `dune install` not performing artifact
+  substitution.
+
```

Comment:
this fixes an issue that hasn't been part of a release, so it shouldn't be mentioned in the final changelog.